kratos/python_scripts/new_linear_solver_factory.py

```diff
@@ -83,12 +78,17 @@ def ConstructSolver(configuration):
         import KratosMultiphysics.EigenSolversApplication
         linear_solver = KratosMultiphysics.EigenSolversApplication.PardisoLUSolver(configuration)
 
+     # emulating the solvers of the MKLSolversApplication through the EigenSolversApplication
+    elif (solver_type == "ParallelMKLPardisoSolver"):
+        import KratosMultiphysics.EigenSolversApplication
```

Comment:
maybe we could even have a message saying DEPRECATED: please use KratosMultiphysics.EigenSolversApplication.PardisoLUSolver

anyway what you did is fine to me (thx)